modules/terrain/src/lib/delatin/index.js

```diff
@@ -0,0 +1,486 @@
+/* eslint-disable complexity, max-params, max-statements, max-depth, no-constant-condition */
```

Comment:
You should copy the license into a comment at the top of this file
https://github.com/mapbox/delatin/blob/master/LICENSE

Reply:
got it